<launch>
<!--<include file="$(find dlib_detector)/launch/3dsensor.launch" />-->
  

  <include file="$(find dlib_detector)/launch/dlib_detector.launch">
	<arg name="image_topic" value="/camera/rgb/image_raw" />
	<arg name="detections_topic" value="/detector/traffic_signs" />
    <arg name="detector" value="$(find dlib_detector)/data/traffic_signs.dat"/>
  </include>

  <node pkg="detection_msgs" type="overlay_visualizer" name="traffic_sign_visualizer" output="screen">
    <remap from="camera" to="/camera/rgb/image_raw" />
    <remap from="detections" to="/detector/traffic_signs" />
    <param name="cleanup_delay" value="0" />
  </node>

  

</launch>
